sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -5895,6 +5934,18 @@ export class BlobLeaseClient {
     options: LeaseOperationOptions = {}
   ): Promise<LeaseOperationResponse> {
     const { span, spanOptions } = createSpan("BlobLeaseClient-changeLease", options.tracingOptions);
+
+    if (
+      this._isContainer &&
+      ((options.conditions?.ifMatch && options.conditions?.ifMatch !== ETagNone) ||
+        (options.conditions?.ifNoneMatch && options.conditions?.ifNoneMatch !== ETagNone) ||
+        options.conditions?.ifTags)
+    ) {
+      throw new RangeError(
+        "the IfMatch, IfNoneMatch and IfTags access conditions must have their default values because they are ignored by the service"
```

Comment:
>their default values [](start = 73, length = 20)

"their default values" sounds confusing. I think the default values are referring to `undefined`.

Reply:
No. For IfMatch and IfNoneMatch their default values can be `ETagNone`.

Reply:
How about
"The IfMatch, IfNoneMatch and IfTags access conditions are ignored by the service. Values other than undefined or their default values are not acceptable."?